tests/suites/test_suite_pkparse.data

```diff
@@ -174,74 +174,290 @@ Parse RSA Key #38 (PKCS#8 encrypted v2 PBKDF2 3DES)
 depends_on:MBEDTLS_DES_C:MBEDTLS_MD_CAN_SHA1:MBEDTLS_PEM_PARSE_C:MBEDTLS_PKCS5_C:MBEDTLS_CIPHER_C:MBEDTLS_CIPHER_MODE_CBC
 pk_parse_keyfile_rsa:"../framework/data_files/rsa_pkcs8_pbes2_pbkdf2_1024_3des.pem":"PolarSSLTest":0
 
+Parse RSA Key #38.1 (PKCS#8 encrypted v2 PBKDF2 AES-128-CBC)
+depends_on:MBEDTLS_AES_C:PSA_WANT_ALG_SHA_256:MBEDTLS_PEM_PARSE_C:MBEDTLS_PKCS5_C:MBEDTLS_CIPHER_C:MBEDTLS_CIPHER_MODE_CBC
```

Comment:
Neither `MBEDTLS_MD_CAN_SHA1` nor `PSA_WANT_ALG_SHA_256` but `MBEDTLS_MD_CAN_SHA256`. Apart from that LGTM